### PR TITLE
Build nodemgr in CI

### PR DESCRIPTION
**Pull request by michal-clapinski**

### CODE REVIEW

**Comment by michal-clapinski** [comment]
recheck no bug

File: CIScripts/Build/BuildFunctions.ps1 [hunk 1]
```diff
@@ -203,7 +203,45 @@ function Invoke-AgentBuild {
...
+            "database:node_mgr",
+            "build/debug/sandesh/common/dist",
+            "sandesh/library/python:pysandesh",
+            "contrail-nodemgr",
+            "vrouter:node_mgr"
+        )
+        $ComponentsString = $Components -Join " "
+
+        Invoke-NativeCommand -ScriptBlock {
+            Invoke-Expression "scons $ComponentsString" | Tee-Object -FilePath $LogsPath/build_nodemgr.log
```
Comment (krdln) [inline CIScripts/Build/BuildFunctions.ps1]: Shouldn't we pass `$BuildMode` here somewhere?

You can use `scons @Components`, like here https://github.com/Juniper/contrail-windows-ci/blob/cc88d3728c344d7b43e72991402138afb7f2ed7b/Test/TestConfigurationUtils.ps1#L502

File: CIScripts/Build/BuildFunctions.ps1 [hunk 2]
```diff
@@ -203,7 +203,45 @@ function Invoke-AgentBuild {
...
+    Param ([Parameter(Mandatory = $true)] [string] $OutputPath,
+           [Parameter(Mandatory = $true)] [string] $LogsPath,
+           [Parameter(Mandatory = $false)] [string] $BuildMode = "debug")
+
+    $Job.PushStep("Nodemgr build")
+
+    $Job.Step("Building nodemgr", {
+        $Components = @(
+            "database:node_mgr",
+            "build/debug/sandesh/common/dist",
```
Comment (krdln) [inline CIScripts/Build/BuildFunctions.ps1]: s/build/`$BuildMode`/
Also, maybe create a PR that adds an alias to scons.
Perhaps an alias to build the contrail-nodemgr with all its dependencies?

**Comment by michal-clapinski** [comment]
recheck no bug

**Comment by michal-clapinski** [comment]
recheck no bug

**Pull request merged**